error TS2420: Class 'NgxMatColorPickerComponent' incorrectly implements interface 'CanColor'

```ERROR in node_modules/@angular-material-components/color-picker/lib/components/color-picker/color-picker.component.d.ts:34:22 - error TS2420: Class 'NgxMatColorPickerComponent' incorrectly implements interface 'CanColor'.
  Property 'defaultColor' is missing in type 'NgxMatColorPickerComponent' but required in type 'CanColor'.

34 export declare class NgxMatColorPickerComponent implements OnInit, OnDestroy, CanColor {
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~

  node_modules/@angular/material/core/common-behaviors/color.d.ts:15:5
    15     defaultColor: ThemePalette | undefined;
           ~~~~~~~~~~~~
    'defaultColor' is declared here.
node_modules/@angular-material-components/datetime-picker/lib/datetime-picker.component.d.ts:52:22 - error TS2420: Class 'NgxMatDatetimePicker<D>' incorrectly implements interface 'CanColor'.
  Property 'defaultColor' is missing in type 'NgxMatDatetimePicker<D>' but required in type 'CanColor'.

52 export declare class NgxMatDatetimePicker<D> implements OnDestroy, CanColor {
                        ~~~~~~~~~~~~~~~~~~~~

  node_modules/@angular/material/core/common-behaviors/color.d.ts:15:5
    15     defaultColor: ThemePalette | undefined;
           ~~~~~~~~~~~~
    'defaultColor' is declared here.

** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **
70% building 951/951 modules 0 active    
    ERROR in node_modules/@angular-material-components/color-picker/lib/components/color-picker/color-picker.component.d.ts:34:22 - error TS2420: Class 'NgxMatColorPickerComponent' incorrectly implements interface 'CanColor'.
      Property 'defaultColor' is missing in type 'NgxMatColorPickerComponent' but required in type 'CanColor'.
    
    34 export declare class NgxMatColorPickerComponent implements OnInit, OnDestroy, CanColor {
                            ~~~~~~~~~~~~~~~~~~~~~~~~~~
    
      node_modules/@angular/material/core/common-behaviors/color.d.ts:15:5
        15     defaultColor: ThemePalette | undefined;
               ~~~~~~~~~~~~
        'defaultColor' is declared here.
    node_modules/@angular-material-components/datetime-picker/lib/datetime-picker.component.d.ts:52:22 - error TS2420: Class 'NgxMatDatetimePicker<D>' incorrectly implements interface 'CanColor'.
      Property 'defaultColor' is missing in type 'NgxMatDatetimePicker<D>' but required in type 'CanColor'.
    
    52 export declare class NgxMatDatetimePicker<D> implements OnDestroy, CanColor {
                            ~~~~~~~~~~~~~~~~~~~~
    
      node_modules/@angular/material/core/common-behaviors/color.d.ts:15:5
        15     defaultColor: ThemePalette | undefined;
               ~~~~~~~~~~~~
        'defaultColor' is declared here.
    
```
Environment:
```
$ ng version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 10.1.0
Node: 14.9.0
OS: linux x64

Angular: 10.1.0
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.1001.0
@angular-devkit/build-angular     0.1001.0
@angular-devkit/build-optimizer   0.1001.0
@angular-devkit/build-webpack     0.1001.0
@angular-devkit/core              10.1.0
@angular-devkit/schematics        10.1.0
@angular/cdk                      10.2.0
@angular/flex-layout              10.0.0-beta.32
@angular/material                 10.2.0
@ngtools/webpack                  10.1.0
@schematics/angular               10.1.0
@schematics/update                0.1001.0
rxjs                              6.6.2
typescript                        3.9.7
webpack                           4.44.1
```
```
$ npm list @angular-material-components/datetime-picker
...
└── @angular-material-components/datetime-picker@4.0.2
```
```
$ npm list @angular-material-components/color-picker
...
└── @angular-material-components/color-picker@4.0.1
```

--- a/projects/color-picker/src/lib/components/color-picker/color-picker.component.ts
+++ b/projects/color-picker/src/lib/components/color-picker/color-picker.component.ts
@@ -113,6 +113,16 @@ export class NgxMatColorPickerComponent implements OnInit, OnDestroy, CanColor {
   get opened(): boolean { return this._opened; }
   set opened(value: boolean) { value ? this.open() : this.close(); }
   private _opened = false;
+  
+  /** Default Color palette to use on the datepicker's calendar. */
+  @Input()
+  get defaultColor(): ThemePalette {
+    return this._defaultColor;
+  }
+  set defaultColor(value: ThemePalette) {
+    this._defaultColor = value;
+  }
+  _defaultColor: ThemePalette = 'primary';
 
   /** Color palette to use on the datepicker's calendar. */
   @Input()

--- a/projects/datetime-picker/src/lib/datetime-picker.component.ts
+++ b/projects/datetime-picker/src/lib/datetime-picker.component.ts
@@ -128,6 +128,16 @@ export class NgxMatDatetimePicker<D> implements OnDestroy, CanColor {
 
   /** The view that the calendar should start in. */
   @Input() startView: 'month' | 'year' | 'multi-year' = 'month';
+  
+   /** Default Color palette to use on the datepicker's calendar. */
+  @Input()
+  get defaultColor(): ThemePalette {
+    return this._defaultColor;
+  }
+  set defaultColor(value: ThemePalette) {
+    this._defaultColor = value;
+  }
+  _defaultColor: ThemePalette = 'primary';
 
   /** Color palette to use on the datepicker's calendar. */
   @Input()

